releasenotes/notes/moved-error-constants-97c7278a381b56d0.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+     Instead of using error constants from ``ddtrace.ext.errors``. Use constants from ``ddtrace.constants`` module. 
+     For Example:: ``ddtrace.ext.errors.ERROR_MSG`` -> ``ddtrace.constants.ERROR_MSG``
+      
+deprecations:
+  - |
+    Moved ``ddtrace.ext.errors`` constants into ``ddtrace.constants`` module. 
```

Comment:
should we also mention removing `get_traceback`? with no upgrade plan?

Reply:
Good catch. I'll add this into the release note